scala/README.md

```diff
@@ -284,13 +327,28 @@ It is definitely recommended to read the full Twitter's "[Effective Scala]" guid
     val answer = ys exists (...)
     ```
 
-0. Avoid wildcard-importing entire packages: `import package._`
+0. Avoid implicit conversions, use implicit value classes ([extension methods](http://docs.scala-lang.org/overviews/core/value-classes.html#extension-methods)) instead. In other words, do not import `scala.language.implicitConversions`:
```

Comment:
I'll have to relax this rule a little because implicit classes cannot replace implicit methods in all situations :-( There are a few use cases where implicit method conversions are still needed.